src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,21 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: (7 * $size-day) + 2 * $padding-month;
```

Comment:
Try not to use "magic values", but to put them in variables, because now it is not clear what 7 and 2 are and whether it can be changed in the future, or put the full result of the calculation in a separate variable.